mungegithub/mungers/release-note-label.go

```diff
@@ -203,7 +205,7 @@ func (r *ReleaseNoteLabel) isStaleIssueComment(obj *github.MungeObject, comment
 	if !obj.IsRobot(comment.User) {
 		return false
 	}
-	if *comment.Body != releaseNoteBody && *comment.Body != parentReleaseNoteFormat {
```

Comment:
!

Reply:
Sorry, what did you mean?

Reply:
Oh, I hadn't seen that you mentioned this error in the PR body. I was surprised to see that the comment body was checked against the format string not the body string.